ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.scss

```diff
@@ -4,356 +4,136 @@
-      width: 100%;
-      height: 20px;
-      cursor: pointer;
-    }
-
-    .rail-inner,
-    .rail-solar {
-      position: absolute;
-      top: 50%;
-      transform: translateY(-50%);
-      height: 4px;
-      pointer-events: none;
-    }
-
-    .rail-inner {
-      width: 100%;
-      background: var(--iui-color-background-backdrop);
-    }
-
-    .rail-solar {
-      background: var(--iui-color-background-accent);
-      z-index: 1;
-    }
-
-    .rail-tooltip {
-      position: absolute;
-      margin-left: -11px;
-      top: 0;
-      transform: translateY(-50%);
-      z-index: 1;
-    }
-
-    .solar-handle {
-      position: absolute;
-      top: 50%;
-      z-index: 4;
-      width: 38px;
-      height: 20px;
-      border-radius: var(--iui-border-radius-1);
-      border: 1px solid $buic-background-5;
-      background-image: linear-gradient(
-          to bottom,
-          $buic-background-3,
-          $buic-background-3
-        ),
-        linear-gradient(to bottom, $buic-background-1, $buic-background-1);
-      display: flex;
-      align-items: center;
-      justify-content: center;
-      box-sizing: border-box;
-      transform: translate(-50%, -50%);
-      cursor: pointer;
-
-      > div {
-        width: 1px;
-        height: 10px;
-        opacity: 0.25;
-        margin: 0 1px;
-        background-color: var(--iui-color-text);
-      }
-    }
-
-    .tick-mark {
-      position: absolute;
-      bottom: calc(#{$bottom-position} + 15px);
-      width: 1px;
-      height: 5px;
-      background: rgb(200, 200, 200);
-    }
-
-    .tick-label {
-      position: absolute;
-      bottom: $bottom-position;
-      font-size: 10px;
-      text-align: center;
-    }
-
-    .sunrise,
-    .sunset {
-      font-size: 22px;
-      cursor: default;
-    }
-
-    .sunset {
-      margin: 0px 16px 2px 24px;
-    }
-
-    .sunrise {
-      color: var(--iui-color-icon-warning);
-      margin: 0 24px 0 8px;
+  @media only screen and (max-width: 490px) {
+    .solar-timeline-date-time {
+      display: none;
     }
   }
 }
 
-.solar-timeline-wrapper > .header > .speed-container:hover > .speed {
-  opacity: 1;
-  width: 80px;
-}
-
-.solar-timeline-wrapper.expanded .header .expanded-icon {
-  transform: rotate(180deg);
-}
-
-.solar-timeline-wrapper.expanded > .header > .current-date {
-  margin-left: 70px;
+.solar-slider {
+  grid-area: solar-timeline;
+  margin-inline: var(--iui-size-2xs);
 }
 
-.solar-timeline-wrapper .solar-timeline-expanded {
-  display: none;
-  margin-bottom: 8px;
-}
-
-.solar-timeline-wrapper .solar-timeline {
+.solar-timeline-start,
+.solar-timeline-end {
   display: flex;
+  align-items: center;
 }
 
-.solar-timeline-wrapper.expanded {
-  height: 100px;
+.solar-timeline-start {
+  grid-area: solar-timeline-start;
 }
 
-.shadow-settings-color-top {
-  display: inline;
+.solar-timeline-end {
+  grid-area: solar-timeline-end;
+  justify-content: flex-end;
 }
+// #endregion Solar timeline
+
+// #region Color picker
```

Comment:
I'd like to swap this out with [the iTwinUI color picker](https://itwinui.bentley.com/docs/colorpicker#advanced), but that can be done in a separate PR.

Reply:
I have swapped out the components in the timeline and deprecated AppUI color picker components.

Reply:
Deprecations are in separate PR: https://github.com/iTwin/appui/pull/755